packages/components/src/components/vertical-tab/vertical-tab-content-container.jsx

```diff
@@ -3,23 +3,46 @@ import React from 'react';
 import { Route, Switch } from 'react-router-dom';
 import Icon from 'Components/icon';
```

Comment:
Can u import the `icon.jsx` file directly since it's a peer package component?
This will prevent the duplicated css since importing the index will cause the `icon.scss` to be imported again.